alert.go

```diff
@@ -0,0 +1,81 @@
+package aapi
+
+import (
+	"fmt"
+	"net/http"
+)
+
+// AlertService handles requests to the on-call alerts endpoint.
+//
+// // https://grafana.com/docs/oncall/latest/oncall-api-reference/alerts/
+type AlertService struct {
+	client *Client
+	url    string
+}
+
+// NewAlertService creates an AlertService with the defined URL.
+func NewAlertService(client *Client) *AlertService {
+	alertService := AlertService{}
+	alertService.client = client
+	alertService.url = "alerts"
+	return &alertService
+}
+
+// PaginatedAlertsResponse represents a paginated response from the on-call alerts API.
+type PaginatedAlertsResponse struct {
+	PaginatedResponse
+	Alerts []*Alert `json:"results"`
+}
+
+// Alert represents an on-call alert.
+type Alert struct {
+	ID           string       `json:"id"`
+	AlertGroupID string       `json:"alert_group_id"`
```

Comment:
Note that I've used all-uppercase acronyms in struct field names, as per [Go convention](https://github.com/golang/go/wiki/CodeReviewComments#initialisms). However, this contrasts a bit with patterns used elsewhere in `amixr-api-go-client`, such as `OnCallShift`'s `TeamId` field.